### PR TITLE
Pfm 2176 banner changes

### PR DESCRIPTION
**Pull request by hariomsinha-eGov**
This PR has changes for : 
1. Banner Integration for response screen.
2. Form Composer enhancements for handling Toasts for Error.

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import { Link, useHistory, useLocation } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { Banner, Card, CardText, LinkLabel, EditIcon} from "@egovernments/digit-ui-react-components";
+import useQueryParams from "../../../../../../libraries/src/hooks/useQueryParams";
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js]: you shouldn't import outside a package.
@hariomsinha-eGov

**Review by jagankumar-egov — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
```
Comment (nipunarora-eGov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js:1]: @hariomsinha-eGov Can we make this component as a generic comp in react-components library. We have multiple response comps present in several modules. We have one generic one in works/src/components/response.js

Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js:1]: yes, we can have that. Also, in the projects case, the banner had the updates to show multiple IDs in diff rows. That change is done in Banner ( one from react components )

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js (hunk 1 above)
Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js]: sure Jagan. I should have used the hook to call the hook.

**Pull request merged**